### PR TITLE
Update dlp-configure-endpoint-settings.md

### PR DESCRIPTION
**Pull request by edwinchan-microsoft**
added required KBs for authorized groups and removed the preview sign up link due to these features moving to full public preview

### CODE REVIEW

**Comment by edwinchan-microsoft** [comment]
@chrfox please approve PR

File: microsoft-365/compliance/dlp-configure-endpoint-settings.md [hunk 1]
```diff
@@ -369,18 +369,16 @@ By default, when devices are onboarded, activity for Office, PDF, and CSV files
 
 File activity will always be audited for onboarded devices, regardless of whether they're included in an active policy.
 
-> [!IMPORTANT]
-> Before you can use [Printer groups (preview)](#printer-groups-preview), [Removable storage device groups](#removable-storage-device-groups-preview), [Network share groups](#network-share-groups-preview), and [VPN settings](#vpn-settings-preview) you must register [here](https://forms.office.com/r/GNVTFvxuZv).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-configure-endpoint-settings.md]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @edwinchan-microsoft, kindly commit the suggested change to remove the extra blank line. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Review by cesarapascual — APPROVED** [review]

**Review by cesarapascual — APPROVED** [review]

**Comment by chrfox** [comment]
back to Edwin chan Contributor license agreement incomplete

**Comment by edwinchan-microsoft** [comment]
> 
@microsoft-github-policy-service agree [company="Microsoft"]

**Comment by edwinchan-microsoft** [comment]
> ```
> @microsoft-github-policy-service agree
> ```

@microsoft-github-policy-service agree

**Pull request merged**